extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -64,26 +81,155 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
+					this.migrationStateModel._databaseBackup.migrationMode = MigrationMode.ONLINE;
+				}
+			}));
+
+		this._offlineButton = this._view.modelBuilder.radioButton()
+			.withProps({
+				label: constants.DATABASE_BACKUP_MIGRATION_MODE_OFFLINE_LABEL,
+				name: buttonGroup,
+				checked: this.migrationStateModel._databaseBackup.migrationMode === MigrationMode.OFFLINE,
+				CSSStyles: { ...styles.LABEL_CSS, 'margin-top': '12px' },
+			}).component();
+		const offlineDescription = this._view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_MIGRATION_MODE_OFFLINE_DESCRIPTION,
+				CSSStyles: { ...styles.NOTE_CSS, 'margin-left': '20px' }
+			}).component();
+		this._disposables.push(
+			this._offlineButton.onDidChangeCheckedState(checked => {
+				if (checked) {
+					this.migrationStateModel._databaseBackup.migrationMode = MigrationMode.OFFLINE;
+				}
+			}));
+
+		const flexContainer = this._view.modelBuilder.flexContainer()
+			.withItems([
+				this._onlineButton,
+				onlineDescription,
+				this._offlineButton,
+				offlineDescription]
+			).withLayout({ flexFlow: 'column' })
+			.component();
+
+		return flexContainer;
+	}
+
+	private createBackupLocationComponent(): azdata.FlexContainer {
+		const buttonGroup = 'networkContainer';
+
+		const selectLocationText = this._view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_PAGE_DESCRIPTION,
+				CSSStyles: { ...styles.BODY_CSS }
+			}).component();
+
+		this._networkShareButton = this._view.modelBuilder.radioButton()
+			.withProps({
+				name: buttonGroup,
+				label: constants.DATABASE_BACKUP_NC_NETWORK_SHARE_RADIO_LABEL,
+				checked: this.migrationStateModel._databaseBackup.networkContainerType === NetworkContainerType.NETWORK_SHARE,
+				CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
+			}).component();
+
+		this._disposables.push(
+			this._networkShareButton.onDidChangeCheckedState(async checked => {
+				if (checked) {
+					this.migrationStateModel._databaseBackup.networkContainerType = NetworkContainerType.NETWORK_SHARE;
+					await utils.updateControlDisplay(this._dmsInfoContainer, true);
+				}
+			}));
+
+		this._blobContainerButton = this._view.modelBuilder.radioButton()
+			.withProps({
+				name: buttonGroup,
+				label: constants.DATABASE_BACKUP_NC_BLOB_STORAGE_RADIO_LABEL,
+				checked: this.migrationStateModel._databaseBackup.networkContainerType === NetworkContainerType.BLOB_CONTAINER,
+				CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
+			}).component();
+
+		this._disposables.push(
+			this._blobContainerButton.onDidChangeCheckedState(async checked => {
+				if (checked) {
+					this.migrationStateModel._databaseBackup.networkContainerType = NetworkContainerType.BLOB_CONTAINER;
+					await utils.updateControlDisplay(this._dmsInfoContainer, false);
+				}
+			}));
+
+		const flexContainer = this._view.modelBuilder.flexContainer()
+			.withItems([
+				selectLocationText,
+				this._blobContainerButton,
```

Comment:
I noticed that we've now switched the order of the options - previously the network share option was first, and the storage account option was second.

Now the order has been switched: 
![image](https://user-images.githubusercontent.com/11844501/207176918-4ce50f7a-32a8-4693-97d9-d0c189bfb73f.png)

But internally in the code here, it seems that the network share scenario is meant to go first (e.g. a few lines above this, the network share radio button is defined before the blob container radio button, etc)

Was this change intentional? If so, would it have any implications on what users would determine the "default" or "recommended" scenario would be? (e.g. seeing the storage account scenario showing up first now, in my head sorta tells me that that's the one I probably want to try first)

Reply:
+@croblesm

Reply:
Hi @raymondtruong, yes this is intentional as the Azure storage blob scenario is the most popular choice in our user base. 

And yes, that is exactly what we are encouraging here. Try the Azure blob scenario first, then the rest :)

Reply:
Got it, thanks @croblesm! I just wanted to make sure it was intentional. Aside from that though I don't see any functional issues surrounding switching them or anything, so resolving this issue for now.